app/modules/tmr.c

```diff
@@ -148,19 +143,13 @@ static int tmr_now(lua_State* L){
 }
 
 static timer_t tmr_get( lua_State *L, int stack ) {
-	// Deprecated: static 0-6 timers control by index.
-	luaL_argcheck(L, (lua_isuserdata(L, stack) || lua_isnumber(L, stack)), 1, "timer object or numerical ID expected");
-	if (lua_isuserdata(L, stack)) {
-		return (timer_t)luaL_checkudata(L, stack, "tmr.timer");
-	} else {
-		uint32_t id = luaL_checkinteger(L, 1);
-		luaL_argcheck(L, platform_tmr_exists(id), 1, "invalid timer index");
-		return &alarm_timers[id];
-	}
-	return 0;
+	timer_t t = (timer_t)luaL_checkudata(L, stack, "tmr.timer");
+	if (t == NULL)
```

Comment:
The documentation at https://www.lua.org/pil/28.2.html disagrees:

    Finally, luaL_checkudata checks whether the object at the given stack position is a userdatum
    with a metatable that matches the given name. It returns NULL if the object does not have the
    correct metatable (or if it is not a userdata); otherwise, it returns the userdata address.